src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -215,6 +215,22 @@
     </PropertyGroup>
   </Target>
 
+  <PropertyGroup>
+    <GetNuGetPackageVersionsDependsOn>$(GetNuGetPackageVersionsDependsOn);CreateVersionInfoFile</GetNuGetPackageVersionsDependsOn>
+  </PropertyGroup>
+  
+  <Target Name="CreateVersionInfoFile"
+          DependsOnTargets="GetLatestCommitHash">
+    <PropertyGroup>
+      <SyncInfoFile>$(PackagesBasePath)\version.txt</SyncInfoFile>
```

Comment:
Did we want to call `version.txt` or `.version`? I personally don't have a preference just curious.